mods/ra/rules/aircraft.yaml

```diff
@@ -457,7 +457,6 @@ U2:
 		SpawnAtLastPosition: False
 		Type: CenterPosition
 		RequiresCondition: enable-smoke
-	RejectsOrders:
```

Comment:
`BADR` still need to reject orders no? `RejectsOrders` was only implemented for `AttackBomber`

Reply:
Yes.